lesson07/assignment/src/views/HowTo.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+<div class="row pb-5">
+  <h1 class="col-12 pt-5 pb-5">How to Cook an Egg: 4 Methods</h1>
+  <section class="col-sm-6">
+  <div class="pb-5" v-for="technique in techniques" v-bind:key="technique.title">
```

Comment:
Similar situation here, you'll want to have one for loop for all of this content. By having an array of techniques, which each technique having its own object `[{title: "sdfsdf", "contentA: "sdfsf", contentB: "sdfsd"}, {title: "sdfsdf", "contentA: "sdfsf", contentB: "sdfsd"}, {title: "sdfsdf", "contentA: "sdfsf", contentB: "sdfsd"}]`

Reply:
This one now seems to be working they way you described. I just have to figure out how to re-style it, but that's a problem for another day. Thanks for letting me correct it!